frontend/src/components/Post/Post.js

```diff
@@ -16,10 +17,9 @@ export default function Post(props) {
 
     useEffect(() => {
         const loadPicture = () => {
-            axios.get(global.config.BACKEND_URL + "/picture/" + props.id).then(
-                (response) => {setPicture(response.data);}
-            ).catch( () => {setPicture(null);}
-            )
+            axios.get(global.config.BACKEND_URL + "/picture/" + props.id)
```

Comment:
As discussed offline, if we could set it to separate lines to keep it consistent with other files